ceph.yaml

```diff
@@ -0,0 +1,145 @@
+    packages:
+      - apache-arrow-dev
+      - autoconf
+      - automake
+      - bash
+      - bison
+      - boost-dev
+      - build-base
+      - cmake
+      - cryptsetup-dev
+      - curl-dev
+      - doxygen
+      - eudev
+      - expat-dev
+      - flex
+      - fuse3-dev
+      - gperf
+      - icu-dev
+      - jq
+      - keyutils-dev
+      - libaio-dev
+      - libcap-dev
+      - libcap-ng-dev
+      - libedit-dev
+      - libnl3-dev
+      - librdkafka-dev
+      - libtirpc-dev
+      - libtool
+      - liburing-dev
+      - libxml2-dev
+      - lttng-ust-tools
+      - lua5.4-dev
+      - lz4-dev
+      - make
+      - nasm
+      - ncurses-dev
+      - ninja
+      - nodejs
+      - npm
+      - oath-toolkit-dev
+      - openldap-dev
+      - openssl-dev
+      - pkgconf
+      - posix-libc-utils
+      - py3.11-cython
+      - py3.11-pyyaml
+      - py3.11-setuptools
+      - py3.11-sphinx-bin
+      - python-3.11-dev
+      - re2-dev
+      - snappy-dev
+      - swagger
+      - systemd-dev
+      - thrift-dev
+      - util-linux-dev
+      - wolfi-base
+      - xfs-scrub
+      - xfsprogs
+      - yaml-cpp-dev
+      - zlib-dev
+      - zstd-dev
+      - apache-arrow-dev
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/ceph/ceph.git
+      tag: v${{package.version}}
+      expected-commit: 58a7fab8be0a062d730ad7da874972fd3fba59fb
+      recurse-submodules: true
+
+  - uses: patch
+    with:
+      patches: fix-submodule.patch
+
+  - runs: |
+      cmake -B build -G Ninja -Wno-dev \
+        -DCMAKE_BUILD_TYPE=Release \
+        -DCMAKE_INSTALL_PREFIX=/usr \
+        -DCMAKE_INSTALL_LOCALSTATEDIR=/var \
+        -DPython3_LIBRARIES=/usr/lib/python3.11 \
+        -DCMAKE_INSTALL_SYSCONFDIR=/etc \
+        -DNINJA_MAX_COMPILE_JOBS=$JOBS \
+        -DALLOCATOR=libc \
+        -DWITH_GTEST_PARALLEL=ON \
+        -DWITH_PYTHON3=3.11 \
+        -DENABLE_GIT_VERSION=OFF \
+        -DLUA_LIBRARIES=/usr/lib/lua5.4/liblua.a \
+        -DWITH_BABELTRACE=OFF \
+        -DWITH_JAEGER=OFF \
+        -DWITH_LTTNG=OFF \
+        -DWITH_MANPAGE=ON \
+        -DWITH_RADOSGW_AMQP_ENDPOINT=OFF \
+        -DWITH_RADOSGW_SELECT_PARQUET=ON \
+        -DWITH_RADOSGW_POSIX=OFF \
+        -DWITH_RDMA=OFF \
+        -DWITH_REENTRANT_STRSIGNAL=ON \
+        -DWITH_SEASTAR=OFF \
+        -DWITH_SPDK=OFF \
+        -DWITH_SYSTEMD=ON \
+        -DWITH_CEPHFS_SHELL=ON \
+        -DWITH_SYSTEM_ARROW=ON \
+        -DWITH_SYSTEM_BOOST=OFF \
+        -DWITH_SYSTEM_FMT=OFF \
+        -DWITH_SYSTEM_LIBURING=ON \
+        -DWITH_SYSTEM_NPM=ON \
+        -DWITH_SYSTEM_ROCKSDB=OFF \
+        -DWITH_SYSTEM_ZSTD=ON \
+        -DWITH_QATLIB=OFF \
+        -DWITH_QATZIP=OFF \
+        -DWITH_TESTS="$(want_check && echo ON || echo OFF)" \
+        -DWITH_THREAD_SAFE_RES_QUERY=ON
+
+  - runs: cmake --build build --target common-options-objs
+
+  - runs: cmake --build build
+
+  - runs: cmake --install build
+
+  - uses: strip
+
```

Comment:
Should we add `-dev` and `-doc` subpackages with tests?

Reply:
Done